lib/ansible/modules/net_tools/cloudflare_dns.py

```diff
@@ -616,22 +617,20 @@ def main():
         ),
         supports_check_mode=True,
         required_if=([
-            ('state', 'present', ['record', 'type']),
-            ('type', 'MX', ['priority', 'value']),
-            ('type', 'SRV', ['port', 'priority', 'proto', 'service', 'value', 'weight']),
-            ('type', 'A', ['value']),
-            ('type', 'AAAA', ['value']),
-            ('type', 'CNAME', ['value']),
-            ('type', 'TXT', ['value']),
-            ('type', 'NS', ['value']),
-            ('type', 'SPF', ['value'])
+            ('state', 'present', ['record', 'type', 'value']),
+            ('state', 'absent', ['record']),
+            ('type', 'SRV', ['proto', 'service']),
```

Comment:
Require condition missing for 'priority' which was present previously.

Reply:
Not needed, for the same reason as it isn't needed for the MX record. Should probably have mentioned that too in the commit message.

Reply:
OK.